fastlane/Fastfile

```diff
@@ -9,6 +9,7 @@ end
 
 platform :ios do
 	lane :test do
+		update_cartfile_with_clent_commit_hash_if_workflow_triggered_by_client
```

Comment:
The code for Fastlane should not be in this PR

Reply:
Yes you're right, my bad I accidentally included here